src/components/ColumnFilter/ColumnFilter.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { SearchLink } from '../SearchLink/SearchLink';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  columnName: string;
+};
+
+export const ColumnFilter: React.FC<Props> = ({ columnName }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+  const formatedColumnName = columnName.toLocaleLowerCase();
+
+  const params: { sort: string | null; order: string | null } = {
+    sort: formatedColumnName,
+    order: sort === formatedColumnName ? 'desc' : null,
```

Comment:
The logic for setting the `order` parameter assumes that if the column is already sorted, it should be in descending order. However, this does not account for toggling between ascending and descending order. Consider initializing `order` as 'asc' when `sort` is not equal to `formatedColumnName` and toggling between 'asc' and 'desc' when `sort` is equal to `formatedColumnName`.